ivy/src/test/scala/sbt/internal/librarymanagement/SftpRepoSpec.scala

```diff
@@ -0,0 +1,38 @@
+package sbt.internal.librarymanagement
+
+import sbt.io._
+import sbt.io.syntax._
+import sbt.util.Level
+import sbt.librarymanagement._
+import sbt.librarymanagement.syntax._
+
+class SftpRepoSpec extends BaseIvySpecification {
```

Comment:
Could we comment out the entire file,  if this is not meant to be run on CI? And perhaps write an instruction on what the person needs to setup in order to use it? Otherwise having access to `~/.ssh/id_rsa` and then doing something with btc-processor looks shady :)